src/NuGet.Core/NuGet.Packaging/PackageExtractor.cs

```diff
@@ -1004,14 +1004,16 @@ private static void DeleteTargetAndTempPaths(string targetPath, string targetTem
                        token,
                        parentId);
 
+                await LogPackageSignatureVerificationAsync(source, package, packageExtractionContext.Logger, verifyResult);
```

Comment:
as discussed, can we please make it for sign packages only? otherwise there'll be lot of noise if we start adding these logs for all the packages.